storage/src/cache/cachedfile.rs

```diff
@@ -379,15 +379,20 @@ impl FileCacheEntry {
                 if self.dio_enabled {
                     self.adjust_buffer_for_dio(&mut buf)
                 }
-                Self::persist_chunk(&self.file, chunk.uncompress_offset(), &buf).map_err(|e| {
-                    eio!(format!(
-                        "do_fetch_chunk failed to persist stargz chunk, {:?}",
-                        e
-                    ))
-                })?;
-                self.chunk_map
-                    .set_ready_and_clear_pending(chunk.as_base())
-                    .unwrap_or_else(|e| error!("set stargz chunk ready failed, {}", e));
+                match Self::persist_chunk(&self.file, chunk.uncompress_offset(), &buf) {
+                    Ok(_) => {
```

Comment:
Seems there's more fundamental issue here:
the chunkmap bits have not been marked as pending for stargz case:(

Reply:
emm,  yeah , I think we need further check and test for the stargz case. But I do not have environment to test it. How about reverting the change for the stargz part, and maybe others can help for this?

Reply:
Could you please help to open an issue for stargz and revert related change? So we could move on with non-stargz part.

Reply:
yes , this should be better